Source/WebCore/page/EventHandler.cpp

```diff
@@ -4438,48 +4435,48 @@ void EventHandler::stopKeyboardScrolling()
         animator->handleKeyUpEvent();
 }
 
-bool EventHandler::beginKeyboardScrollGesture(KeyboardScrollingAnimator* animator, ScrollDirection direction, ScrollGranularity granularity)
+bool EventHandler::beginKeyboardScrollGesture(KeyboardScrollingAnimator* animator, ScrollDirection direction, ScrollGranularity granularity, bool isFirstPhase)
```

Comment:
`isFirstPhase` is not very descriptive. Perhaps better to flip it to be `isKeyRepeat` which is easier to understand.